core/src/test/kotlin/org/stellar/anchor/sep38/Sep38ServiceTest.kt

```diff
@@ -150,43 +156,45 @@ class Sep38ServiceTest {
         null
       )
 
+    val token = createJwtToken("GDJLBYYKMCXNVVNABOE66NYXQGIA5AC5D223Z2KF6ZEYK4UBCA7FKLTG")
+
     // empty sell_asset
-    ex = assertThrows { sep38Service.getPrices(null, null, null, null, null) }
+    ex = assertThrows { sep38Service.getPrices(null, null, null, null, null, null) }
     assertInstanceOf(BadRequestException::class.java, ex)
     assertEquals("sell_asset cannot be empty", ex.message)
 
     // nonexistent sell_asset
-    ex = assertThrows { sep38Service.getPrices("foo:bar", null, null, null, null) }
+    ex = assertThrows { sep38Service.getPrices(token, "foo:bar", null, null, null, null) }
```

Comment:
nit: Setting the token here does not affect the test, right? Can we set them all to `null` so that's obvious?

Reply:
Added `null` token handling.